packages/manager/modules/cloud-connect/src/template.html

```diff
@@ -0,0 +1,155 @@
+                                data-ng-if="step.date"
+                                data-ng-bind="$ctrl.constructor.formatDate(step.date)"
+                            >
+                            </span>
+                        </span>
+                    </span>
+                </span>
+            </ol>
+        </oui-tile-description>
+    </oui-tile-definition>
+</oui-tile>
+
+<oui-tile
+    class="mt-3"
+    data-heading="{{:: 'cloud_connect_services_up' | translate}}"
+>
+    <div class="container-fluid my-3">
+        <oui-datagrid
+            id="{{:: $ctrl.datagridId }}"
+            data-page-size="{{ $ctrl.paginationSize }}"
+            data-rows-loader="$ctrl.loadPage()"
+            data-criteria="$ctrl.criteria"
+            data-on-page-change="$ctrl.onPageChange($pagination)"
+            data-on-criteria-change="$ctrl.onCriteriaChange($criteria)"
+            data-on-sort-change="$ctrl.onSortChange($sort)"
+            data-columns="$ctrl.columnsConfig"
+        >
+            <oui-datagrid-column
+                data-title=":: 'cloud_connect_uuid_label' | translate"
+                data-property="uuid"
+                data-searchable
+                data-filterable
+                data-sortable
+                data-type="string"
+            >
+                <a
+                    data-ng-href="{{ $ctrl.getServiceNameLink($row) }}"
+                    data-ng-bind="$value"
+                ></a>
+            </oui-datagrid-column>
+            <oui-datagrid-column
+                data-title=":: 'cloud_connect_description_label' | translate"
+                data-property="description"
+                data-type="string"
+                data-filterable
+                data-sortable
+            >
+            </oui-datagrid-column>
+            <oui-datagrid-column
+                data-title=":: 'cloud_connect_pop_label' | translate"
+                data-property="pop"
+                data-type="string"
+                data-filterable
+                data-sortable
+            >
+            </oui-datagrid-column>
+            <oui-datagrid-column
+                data-title=":: 'cloud_connect_vrack_label' | translate"
+                data-property="vrack"
+                data-type="string"
+                data-filterable
+                data-sortable
+            >
+            </oui-datagrid-column>
+            <oui-datagrid-column
+                data-title=":: 'cloud_connect_status_label' | translate"
+                data-property="status"
+                data-sortable
+                data-filterable
+            >
+                <span
+                    class="oui-badge"
+                    data-ng-class="{
+                        'oui-badge_success': ($value === 'active'),
+                        'oui-badge_warning': (['creating', 'toCreate', 'toDelete'].includes($value)),
+                        'oui-badge_error': (['deleted', 'suspended'].include($value)),
```

Comment:
**issue(blocking):** We cannot use `.include` from template. Could you please refer to the Controller instead?